db/migrate/20191031190917_add_noneditable_identity_fields_setting.rb

```diff
@@ -0,0 +1,15 @@
+class AddNoneditableIdentityFieldsSetting < ActiveRecord::Migration[5.2]
```

Comment:
Rather than adding this setting via migration, can you add it to `defaults.json`? It can then be imported via the rake task. I think we already have settings changes anyways for this release

Reply:
Everything else looks good!

Reply:
@ying-pbrc can you remove this migration? That way we can make sure that the setting created matches what's listed in `defaults.json`. Migrations are really meant to be for table/column changes rather than creating data.